### PR TITLE
Implement labels, reviewers, upstream branches getters

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: dxcherry/src/github/getters.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { OctokitResponse } from '@octokit/types';
+import { GetResponseDataType, LabelsDataType, ReviewersDataType, BranchesDataType } from './types';
+import { client } from './createClient';
+
+const REPO_DATA = {
+    owner: 'DevExpress',
+    repo: 'DevExtreme'
```
Comment (jtoming830) [inline dxcherry/src/github/getters.ts:7]: This must be in config.

File: dxcherry/src/treeCreator.ts [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
-      new ExtendedTreeItem('19_1'),
-    ]);
+  private static async createBranchesTree(): Promise<ExtendedTreeItem> {
+    const branches = await getBranches();
+
+    return this.createTreeItem(VERSION_ROOT_LABEL, branches);
+  }
+
+  static async createTree(): Promise<ExtendedTreeItem[]> {
+    return [new ExtendedTreeItem('Pull request settings', [
```
Comment (jtoming830) [inline dxcherry/src/treeCreator.ts:30]: Add cache.

Comment (jtoming830) [reply dxcherry/src/treeCreator.ts:30]: Maybe in the future pull requests, but this should be mentioned in the issue.

File: dxcherry/src/treeDataProvider.ts [hunk 3]
```diff
@@ -56,10 +50,10 @@ export class TreeDataProvider implements vscode.TreeDataProvider<ExtendedTreeIte
     this.refresh(element);
   }
 
-  getChildren(element?: ExtendedTreeItem|undefined): vscode.ProviderResult<ExtendedTreeItem[]> {
+  async getChildren(element?: ExtendedTreeItem|undefined): Promise<ExtendedTreeItem[]|undefined> {
```
Comment (jtoming830) [inline dxcherry/src/treeDataProvider.ts:53]: getChildren method's promise should never be resolved with undefined.

File: dxcherry/src/github/getters.ts (hunk 1 above)
Comment (ksercs) [reply dxcherry/src/github/getters.ts:7]: yeap, fixed

File: dxcherry/src/treeCreator.ts (hunk 2 above)
Comment (ksercs) [reply dxcherry/src/treeCreator.ts:30]: I added this issue to checklist, we will fix it later

File: dxcherry/src/treeDataProvider.ts [hunk 4]
```diff
@@ -56,10 +50,10 @@ export class TreeDataProvider implements vscode.TreeDataProvider<ExtendedTreeIte
...
     if (element === undefined) {
-      return this.data;
+      return Promise.resolve(this.tree = await TreeCreator.createTree());
```
Comment (jtoming830) [inline dxcherry/src/treeDataProvider.ts:55]: please no assignments in parameters

**Review by jtoming830 — APPROVED** [review]

File: dxcherry/src/treeDataProvider.ts (hunk 4 above)
Comment (ksercs) [reply dxcherry/src/treeDataProvider.ts]: fixed

**Pull request merged**